packages/star-warp/routes/client.astro

```diff
@@ -0,0 +1,45 @@
+---
+import warpInfo from '@it-astro:star-warp:config';
+
+export const prerender = true;
+---
+
+<script is:inline define:vars={{ warpInfo }}>
+	const search = new URLSearchParams(window.location.search);
+	const query = search.get('q');
+
+	console.log({ warpInfo, search, query });
```

Comment:
is this still supposed to be here?  looks like something from testing...

Reply:
Good catch